avocado/plugins/nrun.py

```diff
@@ -132,7 +132,7 @@ def run(self, config):
         self.spawned_tasks = []  # pylint: disable=W0201
 
         try:
-            if config.get('nrun.spawners.podman.enabled'):
+            if config.get('nrun.spawner') == 'Podman':
```

Comment:
Only think I dislike (it's aesthetics / user experience) is the Capitalization of the spawner names as a requirement from users.  I think lowercase is the way to go.

Reply:
@clebergnu do you mean lowercase in the code and accept any case? or just accept lowercase 'podman' and 'process'?

Reply:
I'd stick with lowercase in the code (parameter option value). I personally stick with a case sensitive check, as I believe most users are OK with the idea of `PrOcEss != process`.